Apsitvarkom.Api/Apsitvarkom.Api.csproj

```diff
@@ -5,6 +5,15 @@
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
     <UserSecretsId>e0e24a99-2f07-4134-a8ca-1b990f6e1a1e</UserSecretsId>
+    <GenerateDocumentationFile>True</GenerateDocumentationFile>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>1701;1702;1591</NoWarn>
```

Comment:
[regarding the first two](https://github.com/dotnet/roslyn/issues/19640) warnings (they came by default).
[CS1591](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs1591) - warning which is not important to us, as we're using the XML docs only for those methods that require additional validation information.